lightkurve/targetpixelfile.py

```diff
@@ -1674,10 +1675,10 @@ def _make_primary_hdu(self, hdu0_keywords):
         # Override the defaults where necessary
         hdu.header['ORIGIN'] = "Unofficial data product"
         hdu.header['DATE'] = datetime.datetime.now().strftime("%Y-%m-%d")
-        hdu.header['TELESCOP'] = "Kepler"
-        hdu.header['CREATOR'] = "lightkurve.KeplerTargetPixelFileFactory"
+        hdu.header['CREATOR'] = "lightkurve.TargetPixelFileFactory"
         hdu.header['OBJECT'] = self.target_id
-        hdu.header['KEPLERID'] = self.target_id
+        if hdu.header['TELESCOP'].lower() == "kepler":
```

Comment:
You'll want to use `hdu.header.get()` here as follows:
```suggestion
        if hdu.header.get('TELESCOP', default='').lower() == "kepler":
```

This way, you will get an empty string instead of an error when the `TELESCOP` keyword is missing.